### ISSUE TITLE
[V14 beta 2] POS PR present in beta V1 and not in beta v2

### ISSUE THREAD

**Issue by bvisible** [frappe/erpnext#30182]
### Information about bug

Hello,
We have installed v14 beta 2 and we found that two PR were present in beta 1 but not in beta 2. Here are the PR : 
#29181
#28298
thank you 

### Module

selling

### Version

Frappe version - v14 beta 2
ERPNext verxion - v14 beta 2

### Installation method

manual install

**Comment by ankush** [comment]
Both PR you've linked are not even merged so how they can be missing from any release? 🤔 

Can you rephrase the problem, maybe I am not getting it right. 😅
